descriptors/sqlserver/debezium-sqlserver-1.9.0.Alpha1.json

```diff
@@ -149,260 +112,13 @@
           "x-name" : "snapshot.mode",
           "x-category" : "CONNECTOR_SNAPSHOT"
         },
-        "snapshot.isolation.mode" : {
```

Comment:
@gunnarmorling I never used SQL Server. How important is that property for SQL Server? include or exclude it?

Reply:
Let's leave it out for now.